presto-cassandra/src/main/java/io/prestosql/plugin/cassandra/CassandraPageSinkProvider.java

```diff
@@ -67,6 +67,6 @@ public ConnectorPageSink createPageSink(ConnectorTransactionHandle transactionHa
                 handle.getTableName(),
                 handle.getColumnNames(),
                 handle.getColumnTypes(),
-                false);
+                handle.isGenerateUuid());
```

Comment:
Not sure what is proper casing according to code-style but `CassandraPageSing` uses `generateUUID`. Can you please unify?

Reply:
Unified to `generateUuid` as we have several `Uuid~` classes.